examples/main.tf

```diff
@@ -48,6 +48,29 @@ resource "instaclustr_cluster" "example" {
     }
   }
 }
+resource "instaclustr_cluster" "gcp_example" {
+  cluster_name = "testclustergcp"
+  node_size = "n1-standard-2"
+  data_centre = "us-east1"
+  sla_tier = "NON_PRODUCTION"
+  cluster_network = "192.168.0.0/18"
+  private_network_cluster = false
+  cluster_provider = {
+    name = "GCP"
+  }
+  rack_allocation = {
+    number_of_racks = 3
+    nodes_per_rack = 1
+  }
+
+  bundle {
+    bundle = "APACHE_CASSANDRA"
+    version = "apache-cassandra-3.11.8.ic2"
```

Comment:
**[Required]** We don't publicly use the legacy version format (part of a change I made for TF provider, it's not 100% but it's the majority). You'll want to change this to `3.11.8`